docs/admin/multiple-zones.md

```diff
@@ -0,0 +1,280 @@
+---
+---
+
+## Introduction
+
+Kubernetes 1.2 adds support for running a single cluster in multiple zones.
+This is a lightweight version of a broader effort for federating multiple
+Kubernetes clusters together (sometimes referred to by the affectionate
+nickname "Ubernetes").  Full federation will allow combining separate
+Kubernetes clusters running in different regions or clouds.  However, many
+users simply want to run a more available Kubernetes cluster in multiple zones
+of their cloud provider, and this is what the multizone support in 1.2 allows
+(we nickname this "Ubernetes Lite").
+
+Multizone support is deliberately limited: a single Kubernetes cluster can run
+in multiple zones, but only within the same region (and cloud provider).  Only
+GCE and AWS are currently supported automatically (though it would be easy to
+add similar support for other clouds or even bare metal, by simply arranging
+for the appropriate labels to be added to nodes and volumes).
+
+
+* TOC
+{:toc}
+
+## Functionality
+
+When nodes are started, the kubelet will automatically add labels to them with
+zone information.
+
+Just as pods in a replication controller or service are spread across nodes,
+the scheduler will use the zone labels to spread first across nodes in
+different zones (via `SelectorSpreadPriority`).  This is a best-effort
+placement, and so if one zone has less available capacity you should not be
```

Comment:
Actually less available capacity is fairly irrelevant here, I think, or at least slightly confusing.  It doesn't enter directly into the scheduling decision.  For example if I have A available capacity in one zone, and 2xA in another, I'm no more likely to land a pod in one or the other zone.  The only relevant thing is whether it is possible to schedule the required number of pods into a given zone to achieve equal spreading across zones.  I suggest rewording to something like:

".. and so if the zones in your cluster are heterogenous (e.g. different numbers of nodes, different types of nodes, or different pod resource requirements), this might prevent perfectly even spreading of your pods across zones.  If desired, you can use homogenous zones (same number and types of nodes) to reduce the probability of unequal spreading."